common/edl/tee.edl

```diff
@@ -14,7 +14,63 @@
 
 enclave
 {
-    include "openenclave/bits/types.h"
+    /**
```

Comment:
Can we move these back to a separate EDL file and just be imported? tee.edl will be going away once all the edl files are published.

Specifically asking this because the header with these structures will be published. The issue is that the _args.h file may end up containing structures which don't need to be included by enclave.h/host.h if it is generated from a file with other structures as arguments. Plus if we want to also put internal structures here, I was the public ones to be separate.

Reply:
I was thinking leaving this part to your PR  :smile:
By the way, the updated PR modifies the oeedge8er such that only user-included (C header) and user-defined structs will be included in _args.h.

Reply:
ah ok, thats fine then :smile: